docs/api/rest_api/rest_api_reference/input/examples/orders/POST/ShipmentCreate.json.example

```diff
@@ -0,0 +1,21 @@
+{
+  "ShipmentCreate": {
+    "identifier": "abc",
+    "ShippingMethod": {
+      "_media-type": "application/vnd.ibexa.api.ShippingMethod+json",
+      "_href": "/api/ibexa/v2/shipping/methods/inpost_parcel_locker"
```

Comment:
```suggestion
      "_href": "/api/ibexa/v2/shipping/methods/parcel_locker"
```
(for reasons mentioned in some other PR earlier)

Reply:
Done in ef48c4ef4c887959331499dc4263621de072fe85